encoding/masking-limiting-etc.md

```diff
@@ -83,22 +126,107 @@ Internally it uses a mask called a Camembert to generate a larger mask
 and limits it to the area affected by a line-darkening script.
 The main mask has no name and is simply dhhmask(mode=3).
 
-Example: Build a simple dehalo mask
+For more information about edgemasks, see [here](https://kageru.moe/blog/article/edgemasks).
+
+#### Example: Build a simple dehalo mask
+
+Suppose you want to remove these halos:
+
+![source](images/halos.png)
+
+![halos](images/src0.png)
+
+(Note that the images shown in your browser are likely resized poorly; 
+you can compare them at full size [here][comparison].)
```

Comment:
This might actually be one of those cases where `here` could be used, but I still prefer a solution without it. Also, comparison is too general of a name. Make this "halo-comparison" and move the url definition closer to its usage, unless it's used multiple times.